### PR TITLE
Zrobione

### PR DESCRIPTION
**Pull request by KonradSadowski**
Mam nadzieję, że wszystko w porządku

### CODE REVIEW

File: models/contacts.js [hunk 1]
```diff
@@ -1,14 +1,122 @@
...
+    console.error("Contact removing error", error)
+    throw error
+  }
+}
+
+
+
+const addContact = async (body) => {
+  try {
+    const contacts = await listContacts();
```
Comment (AdamChodowiecGoIt91) [inline models/contacts.js]: Do you need to fetch contacts before body validation?

File: models/contacts.js [hunk 2]
```diff
@@ -1,14 +1,122 @@
...
+  } catch (error) {
+    console.error("Error adding contact:", error);
+    throw error;
+  }
+}
+
+
+const updateContact = async (contactId, body) => {
+  try {
+    const contacts = await listContacts()
```
Comment (AdamChodowiecGoIt91) [inline models/contacts.js]: same here

File: routes/api/contacts.js [hunk 3]
```diff
@@ -1,25 +1,57 @@
...
-router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
 
 router.delete('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
+  try {
+    const contactId = req.params.contactId
+    const updatedContacts = await contactsModel.removeContact(contactId)
+    res.json({ message: 'Contact deleted successfully', contacts: updatedContacts })
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: could you please always add approtiate status for response?

Comment (KonradSadowski) [reply routes/api/contacts.js]: My bad. Added proper status response.

File: models/contacts.js (hunk 1 above)
Comment (KonradSadowski) [reply models/contacts.js]: I can do fetch after validation. Updated.

File: models/contacts.js (hunk 2 above)
Comment (KonradSadowski) [reply models/contacts.js]: I can do fetch after validation. Updated.

**Pull request merged**